examples/sampling/graphbolt/node_classification.py

```diff
@@ -0,0 +1,331 @@
+import tqdm
+
+
+class SAGE(nn.Module):
+    def __init__(self, in_size, hidden_size, out_size):
+        super().__init__()
+        self.layers = nn.ModuleList()
+        # Three-layer GraphSAGE-mean.
+        self.layers.append(dglnn.SAGEConv(in_size, hidden_size, "mean"))
+        self.layers.append(dglnn.SAGEConv(hidden_size, hidden_size, "mean"))
+        self.layers.append(dglnn.SAGEConv(hidden_size, out_size, "mean"))
+        self.dropout = nn.Dropout(0.5)
+        self.hidden_size = hidden_size
+        self.out_size = out_size
+
+    def forward(self, blocks, x):
+        hidden_x = x
+        for layer_idx, (layer, block) in enumerate(zip(self.layers, blocks)):
+            hidden_x = layer(block, hidden_x)
+            is_last_layer = layer_idx == len(self.layers) - 1
+            if not is_last_layer:
+                hidden_x = F.relu(hidden_x)
+                hidden_x = self.dropout(hidden_x)
+        return hidden_x
+
+
+"""HIGHLIGHT"""
+
+
+def create_dataloader(args, graph, features, itemset, is_train=True):
+    """
+    Get a GraphBolt version of a dataloader for node classification tasks.
+    This function demonstrates how to utilize functional forms of datapipes in
+    GraphBolt.
+    Alternatively, you can create a datapipe using its class constructor.
+    """
+
+    ############################################################################
+    # [Step-1]:
+    # gb.ItemSampler()
+    # [Input]:
+    # 'itemset': The current dataset. (e.g. `train_set` or `valid_set`)
+    # 'args.batch_size': Specify the number of samples to be processed together,
+    # referred to as a 'mini-batch'. (The term 'mini-batch' is used here to
+    # indicate a subset of the entire dataset that is processed together. This
+    # is in contrast to processing the entire dataset, known as a 'full batch'.)
+    # 'is_train': Determining if data should be shuffled. (Shuffling is
+    # generally used only in training to improve model generalization. It's
+    # not used in validation and testing as the focus there is to evaluate
+    # performance rather than to learn from the data.)
+    # [Output]:
+    # An ItemSampler object for handling mini-batch sampling.
+    # [Role]:
+    # Initialize the ItemSampler to sample mini-batche from the dataset.
+    ############################################################################
+    datapipe = gb.ItemSampler(
+        itemset, batch_size=args.batch_size, shuffle=is_train
+    )
+
+    ############################################################################
+    # [Step-2]:
+    # self.sample_uniform_negative()
+    # [Note]:
+    # some extra operations, such as initializing the UniformNegativeSampler for
+    # negative sampling, may be needed in other tasks when it is not training,
+    # but not in node classification. Therefore we omit this step here.
+    ############################################################################
+    if is_train:
```

Comment:
For node classification, this comment can be removed, as `sample_uniform_negative` is outside the scope of this example. Please make the corresponding update to "Step-X".